src/main/java/org/glassfish/grizzly/thrift/client/zookeeper/ZooKeeperConfig.java

```diff
@@ -146,13 +148,8 @@ public long getCommitDelayTimeInSecs() {
 
     @Override
     public String toString() {
-        return "ZooKeeperConfig{" +
```

Comment:
Previous better

Reply:
It's a simple change, so I think it'll be fine.